src/components/Activity/NewRequests.js

```diff
@@ -41,10 +41,14 @@ class NewRequests extends Component {
 
   onRequestSuccess(result) {
     const REQUESTED_PENDING = 2;
-    const REQUESTED_ACCEPT = 1;
 
-    let newRequests = result.requested.filter((value) => value.status === REQUESTED_ACCEPT);
-    newRequests = newRequests.map((value) => value.detail[0]);
+    let newRequests = result.requested.filter((value) => value.status === REQUESTED_PENDING);
+    newRequests = newRequests.map((value) => {
+      value.detail[0]._id = value._id;
+      value.detail[0].contents = value.contents;
+      value.detail[0].request_date = value.request_date;
```

Comment:
Sorry I don't get it. This is for displaying how much time has passed since the mentee's request sented.
e.g) 1 hour ago, 28 days ago